src/main/kotlin/no/nav/helse/flex/reisetilskudd/ReisetilskuddKafkaProducer.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.helse.flex.reisetilskudd
+
+import no.nav.helse.flex.domain.ReisetilskuddSoknad
+import no.nav.helse.flex.kafka.reisetilskuddTopic
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.springframework.stereotype.Component
+
+@Component
+class ReisetilskuddKafkaProducer(
+    private val kafkaProducer: KafkaProducer<String, ReisetilskuddSoknad>
+
+) {
+    fun produserer(reisetilskuddSoknad: ReisetilskuddSoknad) {
```

Comment:
kan vi ikke kalle denne for `send`, eller blir det iffy sånn namespace-messig? presens slik som det er nå føles ikke som riktig grammmatisk form. den riktige blir vel `produser` (imperativ), og det er rart når vi blander substantivet producer og verbet produser

Reply:
kan bytte til send

Reply:
oppdatert